packages/angular/projects/clr-angular/package.json

```diff
@@ -1,8 +1,27 @@
 {
   "name": "@clr/angular",
-  "version": "0.0.1",
+  "version": "@VERSION",
+  "description": "Angular components for Clarity",
+  "homepage": "https://clarity.design",
+  "keywords": ["ng-add", "clarity", "angular", "components"],
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:vmware/clarity.git"
+  },
   "peerDependencies": {
-    "@angular/common": "^10.0.0-rc.4",
-    "@angular/core": "^10.0.0-rc.4"
+    "@angular/common": "^10.0.0",
+    "@angular/core": "^10.0.0",
+    "@clr/core": "@VERSION",
```

Comment:
These line is throwing an error:

```bash
warning " > @clr/angular@4.0.0-rc.1" has unmet peer dependency "@clr/core@4.0.0-rc.1".
warning " > @clr/icons@4.0.0-rc.1" has unmet peer dependency "@webcomponents/custom-elements@^1.0.0".
```

To address this we have two options: 
 1) remove it, cause we do not depend on it, at least for the moment
 2) update the `packages/schematics/src/add/index.ts` to address it when using `ng add @clr/angular` this also requires a change to the `update.ts`

Reply:
Good catch. You're right, we should just remove it for now.